solutions/arithmetic/mean.js

```diff
@@ -0,0 +1,26 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+let numOfStudents = parseInt(readLine());
+let marks = readLine();
+marks = marks.split(" ");
+
+let sumOfMarks = 0;
+
+for(let i = 0; i < numOfStudents; i++){
+    sumOfMarks += parseInt(marks[i]);
+}
+
+let mean = sumOfMarks / numOfStudents;
+
+console.log(Math.floor(mean));
```

Comment:
Instead of `Math.floor(mean) `, I think we can use `parseInt(mean)` for simplicity, though `Math.floor(mean)` works faster.

Reply:
@sailesh-ramayanam @satya-9 @LVSsandeepVarma 

Please your views on this..

Reply:
@Abduliitr  Since  this problem will be given at start of the bootcamp , it's better to go with `parseInt`

Reply:
@LVSsandeepVarma I am ok either way. Using `Math.floor` can be a way to introduce `Math` library.